### PR TITLE
Amend Documents

### PR DESCRIPTION
**Pull request by mtshr**
The pull request fixes some typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@
 
 cder (_see-der_) is a database seeding tool to help you import fixure data in your local environment.
 
-Generating seeds programmatically is a easy task, but maintaining them is not.
+Generating seeds programmatically is an easy task, but maintaining them is not.
 Everytime when your schema is changed, your seed can be broken.
```
Comment (kenkoooo) [inline README.md]:
```suggestion
Every time when your schema is changed, your seed can be broken.
```

File: README.md [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
 It costs your team extra efforts to keep them updated.
```
Comment (kenkoooo) [inline README.md]:
```suggestion
It costs your team extra effort to keep them updated.
```

File: README.md [hunk 3]
```diff
@@ -11,16 +11,16 @@
...
 
 #### with cder you can:
 - maintain your data in a readable format, separated from the seeding program
 - handle reference integrities on-the-fly, using **embedded tags**
-- reuse existing struct and insert function, with only a little glue code is needed
+- reuse existing structs and insert function, with only a little glue code is needed
```
Comment (kenkoooo) [inline README.md]:
```suggestion
- reuse existing structs and insert function,s with only a little glue code is needed
```

File: README.md [hunk 4]
```diff
@@ -250,12 +250,12 @@ Dev:
   email: ${{ ENV(DEVELOPER_EMAIL:-"developer@example.com") }}
 ```
 
-Without specifying the defalut value, all the tags that points to undefined environment vars are simply replaced by empty string "".
+Without specifying the defalut value, all the tags that point to undefined environment vars are simply replaced by empty string "".
```
Comment (kenkoooo) [inline README.md]:
```suggestion
Without specifying the default value, all the tags that point to undefined environment vars are simply replaced by empty string "".
```

File: README.md [hunk 5]
```diff
@@ -250,12 +250,12 @@ Dev:
...
 
 ### Data representation
 cder deserializes yaml data based on [serde-yaml](https://github.com/dtolnay/serde-yaml), that supports powerful [serde serialization framework](https://serde.rs/). With serde, you can deserialize pretty much any struct. You can see a few [sample structs](tests/test_utils/types.rs) with various types of attributes and [the yaml files](tests/fixtures) that can be used as their seeds.
 
-Below shows a few basics of required YAML format.
+Shown below are a few basics of required YAML format.
```
Comment (kenkoooo) [inline README.md]:
```suggestion
Below are a few basics of required YAML format.
```

**Review by kenkoooo — APPROVED** [review]
Thank you for your Pull Request! It looks good to me.

During the code review, I noticed something we can fix, so let me leave some comments.

**Comment by fursich** [comment]
@mtshr Thank you for the thorough check!
Would appreciate if you add a few more corrections that @kenkoooo suggested - all looks fine to me :)

File: README.md (hunk 3 above)
Comment (fursich) [reply README.md]: @kenkoooo  perhaps you meant `s` comes into before comma, right?

```suggestion
- reuse existing structs and insert functions, with only a little glue code is needed
```

**Review by fursich — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (kenkoooo) [reply README.md]: Thanks! Good catch!

File: README.md (hunk 2 above)
Comment (mtshr) [reply README.md]: I fixed as suggested, but may I ask for an explanation in detail for future reference, if you would not mind?
For clarity, just let me tell what I thought: `efforts` here referred to humans' 'actions' required to update seeds, that is, rewriting or adding seeds data and keeping them new, and therefore `[countable] an attempt to do something especially when it is difficult to do` [1] sounded suited to me. 
[1] https://www.oxfordlearnersdictionaries.com/definition/english/effort

File: README.md (hunk 5 above)
Comment (mtshr) [reply README.md]: Okay, let us keep things simple here.

**Comment by mtshr** [comment]
Thank you two for taking a look and @kenkoooo -san for pointing out what I overlooked. I have fixed those as suggested.

While I was reworking I just found a few more I regrettably overlooked to fix. May I ask you to review these too?

- `fixure` to `fixture`
https://github.com/estie-inc/cder/blob/fe54c8f0f3c89eec03bfa80430eba7e655e357db/README.md?plain=1#L12
https://github.com/estie-inc/cder/blob/fe54c8f0f3c89eec03bfa80430eba7e655e357db/README.md?plain=1#L121
https://github.com/estie-inc/cder/blob/fe54c8f0f3c89eec03bfa80430eba7e655e357db/src/database_seeder.rs#L109

- `seed` to `seeds`
https://github.com/estie-inc/cder/blob/fe54c8f0f3c89eec03bfa80430eba7e655e357db/README.md?plain=1#L15

**Comment by mtshr** [comment]
@kenkoooo @fursich  
It seems that `tests/database_seeder_async.rs` and `tests/database_seeder.rs` depend on `tests/test_utils/mod.rs`'s
```rust
pub use mock_database::{sort_records_by_ids, MockTable};
```
but `tests/struct_loader.rs` does not, and therefore clippy failed.

There may be some ways to resolve the issue by like allowing unused imports or removing the above line and specifying only used identities in `tests/*.rs`, but what would be desirable and preferable for the crate? It would be helpful to hear your ideas and policy, or any better way I could not hit upon.

**Review by kenkoooo — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (fursich) [reply README.md]: That makes sense to me (actually my initial intention).
Having said that, it's not a critical point in terms of usage of this crate, so I'd accept either way (please correct if you will!) Thanks again for the detailed review 👍

**Comment by fursich** [comment]
@mtshr 

Thanks for the analysis on CI failure - yes, apparently it's nothing to do with this change (sorry for that..!)

My view on this:
- In principal I'd like to respect the default clippy rules. (avoid `allow` wherever possible)
- I originally imported everything with `*` (which is pretty convenient for tests), but it may not be a good practice.

So my suggestion is to specify all the required imports instead of `*` in all files under `tests/` folder (preferably in a separate PR).
I'll be submitting a PR to fix it, so that you can rebase upon this. Or if you would be willing to create a separate PR, of course I'm happy to accept - let me know!

**Comment by mtshr** [comment]
@fursich 

Thank you for the reply. In order to avoid my misinterpretation, may I request you to make a pull request to resolve the problem? I am grateful for your help.

**Comment by fursich** [comment]
@mtshr 

Looks like it's a false positive (possibly a bug with latest rustc ??) - and I ended up add a workaround.
Anyway the fix is merged, so would you please rebase your PR? Thank you!

**Comment by mtshr** [comment]
@fursich 
Thank you for taking your time to further look into the problem.
I rebased the branch and squashed the commits.

**Review by fursich — APPROVED** [review]
@mtshr All looks great! thanks a lot for contributing to cder 🎉

**Comment by mtshr** [comment]
Thank you for kind supports!

**Pull request merged**